pal/audio/minigame/player-minigame.ts

```diff
@@ -211,6 +221,9 @@ export class AudioPlayerMinigame implements OperationQueueable {
         // KNOWN ISSUES: currentTime doesn't work well
         // on Baidu: currentTime returns without numbers on decimal places
         // on WeChat iOS: we can't reset currentTime to 0 when stop audio
+        if (this._state !== AudioState.PLAYING) {
```

Comment:
Can remove the comments now?

Reply:
Yes, some comments can be removed

Reply:
please ensure the platforms has fixed these known issues
then we can remove this KNOWN ISSUES comments

Reply:
On Baidu Android, the problem stll exist, On Wechat ios, we can reset it manually

Reply:
So this PR can not fix issue on all platforms?

Reply:
> So this PR can not fix issue on all platforms?

No, specifically, there's no bug on Baidu minigame, we can still seek and play. It's just a wrong behavior

Reply:
But can also use cached value event it doesn't have problem. Right?

Reply:
Yes, the reason to cache the seek time, is because on some platform, we cannot seek before play. Even if Baidu return a number without decimal places, it doesn't influence the behavior of the cached time